.github/workflows/feature-pr-status-check.yml

```
@@ -27,7 +27,7 @@ jobs:
       run: |
         $pullRequestNumber = "${{ github.ref }}".Replace("refs/pull/", "").Replace("/merge", "");
         Write-Host "Pull Request Number: $pullRequestNumber";
-        Write-Host "::set-output name=pr-number::$pullRequestNumber";
+        "pr-number=$pullRequestNumber" >> $GITHUB_OUTPUT;
```

This step output is incorrect.
This needs to be done for _**ALL**_ of the workflows.
Update this step output to the following:

```suggestion
        "pr-number=$pullRequestNumber" >> $env:GITHUB_OUTPUT;
```